package.json

```diff
@@ -5,8 +5,11 @@
   "main": "index.js",
   "dependencies": {
     "body-parser": "^1.19.0",
+    "date-and-time": "^0.13.1",
     "dotenv": "^8.2.0",
     "express": "^4.16.2",
-    "mysql": "^2.18.1"
+    "mysql": "^2.18.1",
+    "uuid": "^8.2.0",
+    "uuidv4": "^6.2.0"
```

Comment:
Not a big deal to import both uuid and uuidv4, but I think youre only actually importing uuid in index.js. You might be able to just use uuidv4 since that's the version we want